ethereumetl/cli/stream.py

```diff
@@ -50,8 +50,10 @@
 @click.option('-w', '--max-workers', default=5, show_default=True, type=int, help='The number of workers')
 @click.option('--log-file', default=None, show_default=True, type=str, help='Log file')
 @click.option('--pid-file', default=None, show_default=True, type=str, help='pid file')
+@click.option('--connection-url', default=None, show_default=True, type=str, help='Connection url for file, required for kafka')
```

Comment:
You can also update the description text of the' --output option' to support kafka output

Reply:
Did it, Thanks